packages/flutter/lib/src/semantics/semantics_event.dart

```diff
@@ -86,11 +86,16 @@ class AnnounceSemanticsEvent extends SemanticsEvent {
   /// This property must not be null.
   final TextDirection textDirection;
 
+  /// Determines whether aria announcement needs to be 'polite' or 'assertive' in Flutter web
+  /// Default mode is 'assertive'. Set to false for 'polite'
+  final bool? assertiveAnnouncement;
```

Comment:
Why is this nullable? What's the difference between null and true?

Reply:
Good point.